JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -203,29 +202,29 @@ private data class ParagraphStyling(
 )
 
 private fun ParagraphType.getTextAndParagraphStyle(): ParagraphStyling {
-    var textStyle: TextStyle = +themeTextStyle { body1 }
+    var textStyle: TextStyle = (+MaterialTheme.typography()).body1
```

Comment:
extract `(+MaterialTheme.typography())` to a val. Same for other places in the PR